.storybook/preview-head.html

```diff
@@ -0,0 +1,12 @@
+<style>
+    body {
+        font-family: 'Fira Mono', monospace;
```

Comment:
If this is part of the preview area shouldn't it be populated from theme file configs?